### ISSUE TITLE
Hosted static html analysis page broken

### ISSUE THREAD

**Issue by johnnguyen0215** [relative-ci/bundle-stats#4172]
I've recently encountered an issue where a console error is being thrown on a hosted bundle-stats.html file for my application whenever I try to interact with the page and as a result, the entire page breaks. Here is the console output:

```
Uncaught (in promise) TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
    at z (bundle-stats--bundle-analysis-comparison.html:2:65954)
    at T (bundle-stats--bundle-analysis-comparison.html:2:65233)
    at U (bundle-stats--bundle-analysis-comparison.html:2:66468)
    at bundle-stats--bundle-analysis-comparison.html:2:78472
    at S (bundle-stats--bundle-analysis-comparison.html:2:79773)
    at Object.__ (bundle-stats--bundle-analysis-comparison.html:2:81348)
    at j (bundle-stats--bundle-analysis-comparison.html:2:133832)
    at bundle-stats--bundle-analysis-comparison.html:2:133291
    at Array.filter (<anonymous>)
    at bundle-stats--bundle-analysis-comparison.html:2:133258
```

this can be observed on the demo page as well https://relative-ci.com/bundle-analyzer/bundle-stats/demo/bundle-analysis.

**Comment by vio** [comment]
@johnnguyen0215 thanks for creating the issue! The E2E is missing it, but I can replicate it.

**Comment by vio** [comment]
A transitive dependency(`@floating-ui/dom`) received a new patch version and it is now breaking in Preact. You can use `v4.8.4` until fixed.

**Comment by vio** [comment]
A fix has been released on [`v4.9.1`](https://github.com/relative-ci/bundle-stats/releases/tag/v4.9.1). Feel free to reopen if you still have issues.
